.github/workflows/nightly-build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     if: ${{ needs.verify_commit.outputs.RUN_BUILD == 'true' }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest, ubuntu-latest, macos-latest, macos-13]
```

Comment:
Awesome, thanks. Is that really all we need to do?
At least you need to add the same here:
https://github.com/devonfw/IDEasy/blob/998ef92c86551576eaca1fc8e2f005a67fb8c18d/.github/workflows/release.yml#L13